client/my-sites/plan-features/header/tagline.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ **/
+import { localize } from 'i18n-calypso';
+import {
+	PLAN_FREE,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_PERSONAL
+} from 'lib/plans/constants';
+
+export function PlanFeaturesHeaderTagline( { planType, translate } ) {
+	let tagline;
+
+	switch ( planType ) {
+		case PLAN_FREE:
+			tagline = translate( 'Not sure? Start here' );
+			break;
+		case PLAN_PERSONAL:
+			tagline = translate( 'Best for Bloggers' );
+			break;
+		case PLAN_PREMIUM:
+			tagline = translate( 'Best for Enterpreneurs & Freelancers' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Best for Entrepreneurs &amp; Freelancers' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/illustration-signup-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).